wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx

```diff
@@ -102,6 +103,12 @@ export function Sign() {
                     <BuySellRoll {...signData} />
                   </>
                 );
+              case 'Transaction':
+                return (
+                  <>
+                    <Transaction {...signData} />
+                  </>
+                );
```

Comment:
```suggestion
                return <Transaction {...signData} />
```

Reply:
do you want me to apply these changes to each case where it is present ?

Reply:
You can open a MR specific for that in future, no worries. But keep at least in your code the good one. 👼